kubejs/server_scripts/base/recipes/forge/raw_materials.js

```diff
@@ -0,0 +1,10 @@
+onEvent('recipes', (event) => {
+    materialsToUnify.forEach((material) => {
+        let tagString = `#forge:raw_materials/${material}`;
+        let tag = Ingredient.of(tagString);
+        if (tag && tag?.stacks.size() > 1) {
+            let ore = getPreferredItemInTag(tag);
+            event.shapeless(ore, `#forge:raw_materials/${material}`);
```

Comment:
> - Any ore chunk to craft the preferred ore chunk based on the getPreferredItemInTag function

Why is that desirable? 😅

Reply:
After the change to ore gen, I guess it's unnecessary now :D

Reply:
Alright :D I'd like this removed then please.